/data/sls/u/urop/mvideet/diffusion_reasoning/plot_logits.py:30: FutureWarning: You are using `torch.load` with `weights_only=False` (the current default value), which uses the default pickle module implicitly. It is possible to construct malicious pickle data which will execute arbitrary code during unpickling (See https://github.com/pytorch/pytorch/blob/main/SECURITY.md#untrusted-models for more details). In a future release, the default value for `weights_only` will be flipped to `True`. This limits the functions that could be executed during unpickling. Arbitrary objects will no longer be allowed to be loaded via this mode unless they are explicitly allowlisted by the user via `torch.serialization.add_safe_globals`. We recommend you start setting `weights_only=True` for any use case where you don't have full control of the loaded file. Please open an issue on GitHub for any issues related to this experimental feature.
  logits_list = torch.load(logits_path, map_location=device)  # Load directly to GPU
Traceback (most recent call last):
  File "/data/sls/u/urop/mvideet/diffusion_reasoning/plot_logits.py", line 64, in <module>
    main()
  File "/data/sls/u/urop/mvideet/diffusion_reasoning/plot_logits.py", line 30, in main
    logits_list = torch.load(logits_path, map_location=device)  # Load directly to GPU
  File "/data/sls/scratch/mvideet/anaconda3/envs/diffusion_env/lib/python3.10/site-packages/torch/serialization.py", line 1319, in load
    with _open_file_like(f, "rb") as opened_file:
  File "/data/sls/scratch/mvideet/anaconda3/envs/diffusion_env/lib/python3.10/site-packages/torch/serialization.py", line 659, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/data/sls/scratch/mvideet/anaconda3/envs/diffusion_env/lib/python3.10/site-packages/torch/serialization.py", line 640, in __init__
    super().__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: 'logits_trajectory.pt'
